tasks/elasticsearch-Debian.yml

```diff
@@ -26,3 +26,7 @@
   apt: deb=/tmp/elasticsearch-{{ es_version }}.deb
   when: not es_use_repository
   register: elasticsearch_install_from_package
+
+# ansible uri module requires httplib2
+- name: pip httplib2
+  pip: name=httplib2 extra_args="--user"
```

Comment:
I don't that everybody has pip installed in their production environment. In debian there is [python-httplib2](https://packages.debian.org/fr/sid/python-httplib2).

For info, in Ansible 2.1 (taken from the [uri module doc](http://docs.ansible.com/ansible/uri_module.html#notes)): 

> The dependency on httplib2 was removed in Ansible 2.1